doc/build/changelog/changelog_11.rst

```diff
@@ -163,7 +163,7 @@
     :version: 1.1.15
     :released: November 3, 2017
 
-    .. change:
+    .. change::
         :tags: bug, sqlite
         :tickets: 4099
```

Comment:
These changes (with `.. change:`) were [not shown](https://docs.sqlalchemy.org/en/14/changelog/changelog_11.html) in the changelog 1.1. Maybe this is not a mistake, and these tickets (4099, 4096 and 4098) were missed on purpose?